Quake/sv_main.c

```diff
@@ -706,11 +706,26 @@ void SV_WriteEntitiesToClient (edict_t	*clent, sizebuf_t *msg)
 			continue;
 		//johnfitz
 
+		eval_t *valscale;
```

Comment:
Please don't 'lazy-declare' to keep it C90 compatible. Something like the following should do it:
```diff
diff --git a/Quake/sv_main.c b/Quake/sv_main.c
index f508c2b..db9fc1e 100644
--- a/Quake/sv_main.c
+++ b/Quake/sv_main.c
@@ -603,6 +603,7 @@ void SV_WriteEntitiesToClient (edict_t	*clent, sizebuf_t *msg)
 	vec3_t	org;
 	float	miss;
 	edict_t	*ent;
+	eval_t	*val;
 
 // find the client's PVS
 	VectorAdd (clent->v.origin, clent->v.view_ofs, org);
@@ -695,7 +696,6 @@ void SV_WriteEntitiesToClient (edict_t	*clent, sizebuf_t *msg)
 		if (pr_alpha_supported)
 		{
 			// TODO: find a cleaner place to put this code
-			eval_t	*val;
 			val = GetEdictFieldValue(ent, "alpha");
 			if (val)
 				ent->alpha = ENTALPHA_ENCODE(val->_float);
@@ -706,10 +706,10 @@ void SV_WriteEntitiesToClient (edict_t	*clent, sizebuf_t *msg)
 			continue;
 		//johnfitz
 
-		eval_t *valscale;
-		if (valscale = GetEdictFieldValue(ent, "scale"))
+		val = GetEdictFieldValue(ent, "scale");
+		if (val)
 		{
-			float vs = valscale->_float;
+			float vs = val->_float;
 			if (vs > 0.001f)
 				ent->scale = vs * 16;
 			else
```